### ISSUE TITLE
IntelliSense cannot insert StringComparer when using the short constructor new()

### ISSUE THREAD

**Issue by OnSive** [meziantou/Meziantou.Analyzer#240]
**Context**
Rule identifier: _MA0002_
Target Framework: _.NET 5.0_
Language version: _9.0_

**Describe the bug**
If you're using the `= new();` initialization and not passing a string comparer, the you are prompted to let it be added automatically with IntelliSense.

The problem is, that it cannot add an string comparer probably because of the `new()` keyword.

**To Reproduce**
1. Copy & Paste the code
  ````c#
  HashSet<string> myHashSet = new();
  ````
2. Wait for the IntelliSense bulp
3. Select `Add String Comparer...`
4. Nothing happens

**Additional Information**
I'm currently using code rush which builds up on IntelliSense and adds more features. Because of that it shouldn't make a difference for this bug.

Priority: Lowest ;)

**Comment by meziantou** [comment]
Thanks for reporting the issue! The version [1.0.663](https://www.nuget.org/packages/Meziantou.Analyzer/1.0.663) should fix this bug
